src/topology/test/mod.rs

```diff
@@ -23,3 +29,794 @@ mod doesnt_reload;
 
 mod backpressure;
 mod compliance;
+
+#[cfg(test)]
+mod tests {
```

Comment:
Is there any point to putting effectively this whole file in a submodule? The file itself is already guarded with `#[cfg(test)]` in `src/topology/mod.rs`

Reply:
Hmmm, yeah, that's a fair point. I'll hoist it up and out of the nested module.

Reply:
Addressed in d14568bc3.